romtime/src/otp.rs

```diff
@@ -378,41 +378,41 @@ impl Otp {
         Ok(fuses)
     }
 
-    pub(crate) fn burn_lifecycle_tokens(&self, tokens: &LifecycleHashedTokens) -> McuResult<()> {
+    pub fn burn_lifecycle_tokens(&self, tokens: &LifecycleHashedTokens) -> McuResult<()> {
```

Comment:
Should we move this function back to the `mcu-rom-common` crate because it was specifically limited to it previously?

Reply:
I think this is okay since we might actually want to call this from non-ROM places, like a manufacturing firmware.